setup.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-unused-vars */
+import 'dotenv/config';
+
+import { User } from './src/models/user.js';
+import { Token } from './src/models/token.js';
+import { client } from './src/utils/db.js';
+
+client.sync({ force: true });
```

Comment:
Using 'force: true' in client.sync() will drop and recreate all tables, leading to data loss. Ensure this is intended, especially in a production environment. Consider using 'alter: true' for safer schema updates.